packages/client/hmi-client/src/components/widgets/tera-input-number.vue

```diff
@@ -16,6 +16,7 @@
 				@focusout="$emit('focusout', $event)"
 				type="text"
 				:placeholder="placeholder"
+				@keydown="handleArrowKeys"
```

Comment:
We need to double-check if we can support this (eg where tera-input-number is being used), `handleArrowKeys` eventually calls `updateNumber` which emits an update event. We have had issues where we updated too often that we run into synchronization problems or performance problems.

You may want to add a debouncer here just in case

Reply:
Done